components/src/core/drawer/drawer-body/nav-item/drawer-nav-item.component.ts

```diff
@@ -117,6 +120,7 @@ export class DrawerNavItemComponent extends StateListener implements Omit<Drawer
     @Input() statusColor: string;
     @Input() subtitle: string;
     @Input() title: string;
+    @Input() isHidden = false;
```

Comment:
correct me if I'm wrong, but I don't think we typically use the `is_` prefix for attributes/props/Inputs. I would go for `hidden` instead if it were me.

Reply:
I've removed the `is` part of the `isHidden`